picard/script/__init__.py

```diff
@@ -109,7 +140,7 @@ def enabled_tagger_scripts_texts():
     config = get_config()
     if not config.setting['enable_tagger_scripts']:
         return []
-    return [(s_name, s_text) for _s_pos, s_name, s_enabled, s_text in config.setting['list_of_scripts'] if s_enabled and s_text]
+    return [(s.name, s.content) for s in iter_tagging_scripts_from_config(config=config) if s.enabled and s.content]
```

Comment:
Not directly related to this PR, but the `enabled_tagger_scripts_texts` function could be a generator. It only is used for iterating (as the doc comment already indicates).

Reply:
I did that in https://github.com/metabrainz/picard/pull/2496/commits/a00454afdf5e241c26210f3101da81412c92637b

I changed name to `iter_active_tagging_scripts()` as it yields enabled and non-empty TaggingScriptSetting objects now.